package/pom.xml

```diff
@@ -34,6 +34,11 @@
     <dependencies>
         <dependency>
             <groupId>io.confluent</groupId>
+            <artifactId>build-tools</artifactId>
+            <version>3.3.1-SNAPSHOT</version>
```

Comment:
common depends on build-tools, so we end up with circular refs if i try to do anything smarter here :/